test/facility-investor/get-facility-investors-by-facility-identifier.api-test.ts

```diff
@@ -0,0 +1,131 @@
+import { PROPERTIES } from '@ukef/constants';
+import { withAcbsAuthenticationApiTests } from '@ukef-test/common-tests/acbs-authentication-api-tests';
+import { IncorrectAuthArg, withClientAuthenticationTests } from '@ukef-test/common-tests/client-authentication-api-tests';
+import { withFacilityIdentifierUrlParamValidationApiTests } from '@ukef-test/common-tests/request-url-param-validation-api-tests/facility-identifier-url-param-validation-api-tests';
+import { Api } from '@ukef-test/support/api';
+import { ENVIRONMENT_VARIABLES, TIME_EXCEEDING_ACBS_TIMEOUT } from '@ukef-test/support/environment-variables';
+import { GetFacilityInvestorGenerator } from '@ukef-test/support/generator/get-facility-investor-generator';
+import { RandomValueGenerator } from '@ukef-test/support/generator/random-value-generator';
+import nock from 'nock';
+
+describe('GET /facilities/{facilityIdentifier}/investors', () => {
+  const valueGenerator = new RandomValueGenerator();
+  const facilityIdentifier = valueGenerator.facilityId();
+  const portfolioIdentifier = PROPERTIES.GLOBAL.portfolioIdentifier;
```

Comment:
we can deconstruct this into:
```
const { portfolioIdentifier } = PROPERTIES.GLOBAL;
```

Reply:
Changed in this module and couple other places, but old code style is used in more places, we can replace them later.